packages/terra-tag/src/index.js

```diff
@@ -0,0 +1,7 @@
+import TagList from './TagList';
+import RollupTag from './RollupTag';
+import TagsUtils from './TagsUtils';
+import Tag from './Tag';
+
+/* eslint-disable object-curly-newline */
+export { TagList, RollupTag, TagsUtils, Tag };
```

Comment:
`Tag` should be a default export, otherwise this is a _breaking change_!
```suggestion
export default Tag;
export { TagList, RollupTag, TagsUtils};
```

Reply:
changes done. Thanks